taxcalc/functions.py

```diff
@@ -501,46 +516,31 @@ def StdDed(DSI, _earned, STD, p04470, e00100, e60000,
     if (MARS == 3 or MARS == 6) and (MIDR == 1):
         _standard = 0.
 
-    return (_standard, c04200, _numextra, c15200, c15100, x04500, _txpyers,
-            c04100)
+    return (_standard, c04200, _numextra, c15200, c15100, x04500, c04100)
 
 
 @iterate_jit(nopython=True, puf=False)
-def TaxInc(c00100, c04470, _standard, e37717, c21060, c21040,
-           p04470, c04200, c04500, c04600, x04500,
-           e04805, t04470, f6251, _exact, _feided, c04800, MARS,
+def TaxInc(c00100, _standard, e37717, c21060, c21040, c04500, c04600,
+           e04805, _feided, c04800, MARS,
            II_rt1, II_rt2, II_rt3, II_rt4,
            II_rt5, II_rt6, II_rt7, II_brk1, II_brk2, II_brk3,
-           II_brk4, II_brk5, II_brk6, _numextra, FDED, c02700):
-
-    # c04500 = c00100 - max(c04470, max(c04100, _standard + e37717))
+           II_brk4, II_brk5, II_brk6, c02700):
+    """
+    TaxInc function: ...
+    """
     c04500 = max(0, c00100 - max(c21060 - c21040, _standard + e37717))
 
-    # Check with Dan whether this is right!
-    # if c04470 > _standard:
```

Comment:
I also just heard from @feenberg that he has been testing the SAS code on 2012 and 2013 data as well as 2011 data at IRS. This is good news because there have not been major tax-law changes since 2013, but there have been since 2011.